test/system/data_exports_test.rb

```diff
@@ -173,15 +173,15 @@ def setup
     # processing
     visit data_export_path(@data_export2)
 
```

Comment:
Is there a new test to see is the page is updated after the data export is updated?

Reply:
I've been trying to research a way to do this but haven't been able to get anything to work (time travels, waits etc.), short of page refresh/revisit which defeats the whole purpose. Did you have any ideas?

Reply:
My thought was you could visit the page, update & save the data export, then verify the page auto refreshed. Like how we test logidze, but I haven't tried it myself. My apologies if it can't be done.

Reply:
Talked to Eric since I couldn't figure out a way to do this through the UI, but he suggested testing for the turbo_stream broadcasts which is what will update the UI.

I've added the tests to my `index` PR since that's where I've made the model changes to `broadcast_refreshes`

Reply:
Sounds good.